alc/alconfig.cpp

```diff
@@ -272,7 +272,7 @@ void LoadConfigFromFile(std::istream &f)
                 ConfOpts.erase(ent);
         }
         else if(!value.empty())
-            ConfOpts.emplace_back(ConfigEntry{std::move(fullKey), expdup(value.c_str())});
+            ConfOpts.emplace_back(std::move(fullKey), expdup(value.c_str()));
```

Comment:
Unfortunately this doesn't work if the struct doesn't have a constructor that takes the given arguments.

Reply:
removed.